index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  configs: {
+    tslint: require('./lib/config/all.js'),
```

Comment:
@lemonmade For the consumer to extend the config, they'll have to create a `tslint.json` file in the root of the project with the following: 
```
{
  "extends": "tslint-config-shopify"
}
```
See here https://palantir.github.io/tslint/usage/tslint-json/